app/controllers/application_controller.rb

```diff
@@ -1583,6 +1583,7 @@ def get_view(db, options = {})
       :association           => association,
       :filter                => get_view_filter(options[:filter]),
       :sub_filter            => get_view_process_search_text(view),
+      :supports_filter       => options[:supports_filter],
```

Comment:
yeah I agree with @durandom ... :supports_filter would unintentionally collide with a future feature *named* "filter", which would generate the dynamic method `supports_filter?`... and confusion would ensue.  I like `:supported_features_filter`